entrypoint.sh

```diff
@@ -36,11 +36,14 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+# Set Maven download logging to warn level
+export MAVEN_OPTS=-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+
 # Build Fosstars
 git clone https://github.com/SAP/fosstars-rating-core && \
     cd fosstars-rating-core && \
     git checkout $FOSSTARS_VERSION && \
-    mvn package -DskipTests && \
+    mvn package -ntp -DskipTests && \
```

Comment:
As you updated this line, could you please add a couple of options for skipping unnecessary steps?

```suggestion
    mvn package -ntp -DskipTests -Dcheckstyle.skip -Dmaven.javadoc.skip
 && \
```

Reply:
Thanks for the hint, have updated everything accordingly.